static/docs/commands-reference/install.md

```diff
@@ -35,12 +35,23 @@ is a reminder to run `dvc commit`.
 The installed Git hook automates reminding the user to run either `dvc repro` or
 `dvc commit`.
 
+**Push** Sharing data across environments and making changes to remote storage
+is done by this command. This command searches for changed .dvc files and stages
+them for consultation. Using checksums it finds for existing entries in the
+remote cache. After that it uploads the cache files to the remote.
+
+The installed Git hook automates reminding `dvc push` and corresponds to the 
```

Comment:
probably it's not about reminding, it's about actually executing `dvc push` when you run `git push, exactly like you described below (@pared can confirm this)

Reply:
yes,corrected it.